eng/testing/WasmRunnerAOTTemplate.sh

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env bash
 
-EXECUTION_DIR=$(dirname $0)
-SCENARIO=$3
+# SetCommands defined in eng\testing\tests.wasm.targets
+[[SetCommands]]
+[[SetCommandsEcho]]
 
-cd $EXECUTION_DIR
+EXECUTION_DIR=$(dirname $0)
+if [[ -n "$3" ]]; then
+	SCENARIO=$3
+fi
 
-if [ -z "$HELIX_WORKITEM_UPLOAD_ROOT" ]; then
+if [[ -z "$HELIX_WORKITEM_UPLOAD_ROOT" ]]; then
```

Comment:
What is the reason to use `[[ ... ]]` here instead of `[ ... ]` and in the other places?

Reply:
As far as I could tell `[[` is build-in command of bash, and `[` is alias for `test` binary. It should behave same way in this case.

Reply:
It is less portable, so I was curious if there's reason to use it. I guess we can change it again when needed.